engine/blan19/src/commit/classifier.ts

```diff
@@ -0,0 +1,23 @@
+import { CommitKeywordMapper } from "./mapper/commitKeywordMapper";
+
+export const classifierOfKeyword = (message: string) => {
+  for (const key in CommitKeywordMapper) {
+    const keyword = new RegExp(`${key}`, "i").exec(message);
```

Comment:
```suggestion
    const keyword = new RegExp(`${key}`, "i").exec(message);
```
[질문]
CommitKeywordMapper key type은 string이라서 그냥 key로 적어도 동작할 것 같은데, `${key}`이건 혹시 모를 경우를 대비해서 방어코딩 해주신 건가요? 아니면 오류가 나는건가요?
```ts
interface CommitKeywordMapper {
    [key: string]: Commit;
  }
```

Reply:
맞습니다! key type은 어차피 string이라 템플릿 리터럴로 굳이 형변환 할 필요가 없는데 제 무의식에서 저런 방어코드를 쓴 거 같아요 ㅋㅋ
의미 없는 템플릿 리터럴 제거해서 다시 푸시했어요!

Reply:
앗 과하지 않은 방어코드였다고 생각하긴 했는데! 수정해주셨군요..! 무튼 코드 잘봤습니다 🙂 👍🏻